sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -726,11 +726,14 @@ class JDBCSuite extends SparkFunSuite
 
   test("OracleDialect jdbc type mapping") {
     val oracleDialect = JdbcDialects.get("jdbc:oracle")
-    val metadata = new MetadataBuilder().putString("name", "test_column").putLong("scale", -127)
-    assert(oracleDialect.getCatalystType(java.sql.Types.NUMERIC, "float", 1, metadata) ==
-      Some(DecimalType(DecimalType.MAX_PRECISION, 10)))
     assert(oracleDialect.getCatalystType(java.sql.Types.NUMERIC, "numeric", 0, null) ==
       Some(DecimalType(DecimalType.MAX_PRECISION, 10)))
+    val metadata1 = new MetadataBuilder()
```

Comment:
Could you explain the scenario that could build an empty metadata and pass it to `getCatalystType`?